Djikstra/Djikstra/Headers.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include <iostream>
+#include <cstdlib>
+#include <fstream>
+#include <map>
+
+
+namespace Graph
+{
+	struct Edge
+	{
+		int source_id;
+		int target_id;
+		float distance;
+	};
+
+	struct Vertex
+	{
+		std::string name;
+		std::vector <Edge> edges;
+		int parent_id = -1;
+		bool visited = 0;
+	};
+
+	class Graph
+	{
+	public:
+		std::vector <Vertex> m_vertex;
+
+		bool node_exist( std::string name );
```

Comment:
`std::string` should be passed as `const std::string&`, otherwise a copy is made - which we don't want in this context.